frontend/public/texts/project_texts.js

```diff
@@ -773,7 +773,7 @@ export default function getProjectTexts({ project, user, url_slug, locale }) {
       ),
     },
     title_with_explanation_and_example: {
-      en: "Title (Use a short, english title, e.g. 'Generating energy from ocean waves')",
+      en: "Title (Use a short, compelling title, e.g. 'Generating energy from ocean waves')",
```

Comment:
Mentioning "English" twice seemed redundant -- so I added "compelling" and removed the InfoIcon bit that was under the form. This simplifies this screen and improves the UI IMO. 

<img width="851" alt="Screen Shot 2021-07-25 at 11 43 36 AM" src="https://user-images.githubusercontent.com/1530684/126909925-d39fa70d-acaa-4d42-826f-e7c70cf5e718.png">